app/javascript/components/SponsorsTile.jsx

```diff
@@ -2,17 +2,41 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Card from './Card';
 
+import Stripe from 'images/sponsors/stripe.svg';
+import Shopify from 'images/sponsors/shopify.svg';
+import Contentful from 'images/sponsors/contentful.svg';
+
 import 'stylesheets/sponsors-tile';
 
-const SponsorsTile = ({ sponsors }) => (
-    <Card className="sponsors-tile">
-        <div className="sponsors-tile-title">Our Sponsors</div>
-        <div className="sponsors-tile-img-container">
-            {sponsors.map((sponsor) => (
-                <img key={sponsor.id} src={sponsor.tile_image} alt={sponsor.tile_image_alt} />
-            ))}
-        </div>
-    </Card>
+const sponsors = [
```

Comment:
I moved the info about the sponsors into this component so it could be used on the sponsor page as well as the home page.